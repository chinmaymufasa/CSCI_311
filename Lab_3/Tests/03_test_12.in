2
140
5
2
190
5
2
190
5
2
75
5
2
51
5
2
188
5
2
120
5
2
92
5
2
124
5
2
168
5
2
47
5
2
60
5
2
189
5
2
155
5
2
60
5
2
160
5
2
85
5
2
178
5
2
138
5
2
42
5
-1
