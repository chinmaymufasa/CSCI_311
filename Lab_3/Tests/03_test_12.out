140 
140 190 
140 190 190 
75 140 190 190 
51 75 140 190 190 
51 75 140 188 190 190 
51 75 120 140 188 190 190 
51 75 92 120 140 188 190 190 
51 75 92 120 124 140 188 190 190 
51 75 92 120 124 140 168 188 190 190 
47 51 75 92 120 124 140 168 188 190 190 
47 51 60 75 92 120 124 140 168 188 190 190 
47 51 60 75 92 120 124 140 168 188 189 190 190 
47 51 60 75 92 120 124 140 155 168 188 189 190 190 
47 51 60 60 75 92 120 124 140 155 168 188 189 190 190 
47 51 60 60 75 92 120 124 140 155 160 168 188 189 190 190 
47 51 60 60 75 85 92 120 124 140 155 160 168 188 189 190 190 
47 51 60 60 75 85 92 120 124 140 155 160 168 178 188 189 190 190 
47 51 60 60 75 85 92 120 124 138 140 155 160 168 178 188 189 190 190 
42 47 51 60 60 75 85 92 120 124 138 140 155 160 168 178 188 189 190 190 
